WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -60,6 +66,13 @@ final class BlogDashboardViewController: UIViewController {
         activityIndicatorView.stopAnimating()
     }
 
+    func update(blog: Blog) {
+        self.blog = blog
+        viewModel.blog = blog
+        viewModel.applySnapshotForInitialData()
+        viewModel.start()
```

Comment:
What do you think of renaming these viewModel methods?
```suggestion
        viewModel.clearSnapshot()
        viewModel.loadCards()
```

Reply:
Done in 3cb7f33e994166bca58ebc151382f4baf54fe34c

FYI, `applySnapshotForInitialData` is going to change in the next PR (due to the persistence). I'll update the name there.